db/migrate/20150419001730_create_pieces.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePieces < ActiveRecord::Migration
+  def change
+    create_table :pieces do |t|
+
```

Comment:
the pieces model is missing the piece_name, color, x_coord, y_coord and game_id fields